linters/buildifier/buildifier.test.ts

```diff
@@ -1,5 +1,32 @@
-import { linterCheckTest, linterFmtTest } from "tests";
+import path from "path";
+import { customLinterFmtTest, linterCheckTest } from "tests";
+import { TrunkDriver } from "tests/driver";
+import { TEST_DATA } from "tests/utils";
+
+const preFmt = (driver: TrunkDriver) => {
+  const buildifierTablesContents = `
+{
+  "IsLabelArg": {},
+  "LabelDenylist": {},
+  "IsSortableListArg": {
+    "fizz": true
+  },
+  "SortableAllowlist": {},
+  "NamePriority": {}
+}
+  `;
+
+  driver.moveFile(".buildifier.json", ".trunk/configs/.buildifier.json");
+  driver.deleteFile(".buildifier-tables.json");
+  driver.writeFile(".trunk/configs/.buildifier-tables.json", buildifierTablesContents);
```

Comment:
Not sure how this should be adjusted: should this test go via `.buildifier-tables.json` instead?

Reply:
Inside of `.trunk/configs` should be fine, if that's what you're asking. If you meant something else, we can talk more offline